tests/asyncapi-2.0/Server Variable Object/valid-empty-object.yaml

```diff
@@ -17,7 +17,7 @@ channels:
 
 servers:
   production:
-    url: '{username}.gigantic-server.com:{port}'
+    url: '{username}.gigantic-server.com'
```

Comment:
this test case is for validation if an empty variable object is allowed and not if it is fine to not have a corresponding variable object for a variable